src/test/java/com/learning/selenium/pages/GoogleCloudSearchResultsPage.java

```diff
@@ -0,0 +1,34 @@
+package com.learning.selenium.pages;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+import java.util.List;
+
+public class GoogleCloudSearchResultsPage extends BasePage {
+
+    public GoogleCloudSearchResultsPage(WebDriver driver) {
+        super(driver);
+    }
+
+    @Override
+    protected void open() {
+    }
+
+    @FindBy(css = "[class=gsc-resultsbox-visible]")
+    private WebElement searchResultsBlock;
+
+    @FindBy(css = "a.gs-title b")
+    private List<WebElement> searchResultsList;
+
+    public void searchResultClick(String text) {
+        waitUntilElementClickable(searchResultsBlock);
+        for (WebElement el : searchResultsList) {
+            if (el.getText().equals(text)) {
```

Comment:
Use search using xpath and text?

Reply:
Thanks. It was not good idea. Updated in Task2